src/main/resources/application.properties

```diff
@@ -60,6 +60,8 @@ email.subject: Email Subscription Notification
 notification.failAttempt: 3
 notification.ttl.value: 600
 
+ldap.enabled: false
```

Comment:
Why you adding new ldap.enabled property if there is already one at the line 74?